docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Database Access with Microsoft SQL Server with Active Directory PKINIT authentication (Preview)
+description: How to configure Teleport Database Access with Microsoft SQL Server with Active Directory PKINIT authentication.
+---
+
+<Admonition type="warning" title="Preview">
+  Database Access for Microsoft SQL Server is currently in Preview mode.
+</Admonition>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up access to SQL Server using Active Directory PKINIT authentication.
+- Connect to SQL Server through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access SQL Server Self-Hosted](../../../img/database-access/guides/sqlserver_selfhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access SQL Server Cloud](../../../img/database-access/guides/sqlserver_cloud.png)
+</ScopedBlock>
+
+This guide will focus on SQL Servers using self-hosted Active Directory
+authentication.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A SQL Server database with Active Directory authentication enabled.
+- A SQL Server network listener configured with a Certificate using Subject Alternative Names
+- A Windows machine joined to the same Active Directory domain as the database.
+- A Linux node with network access to the Active Directory.
```

Comment:
- Is "the Active Directory" the correct usage here? Would it make more sense to say, "The Active Directory server" or something similar?
- We mention that the Linux node needs to be able to reach Active Directory, but we don't indicate any requirements for Active Directory itself (only Active Directory authentication for SQL Server)

Reply:
Maybe "An Active Directory installation" or something similar?